airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceDatatypeTest.java

```diff
@@ -612,6 +592,29 @@ protected void addJsonbArrayTest() {
             .build());
   }
 
+  protected void addMoneyTest() {
```

Comment:
can we call it `currency`? hehe

Reply:
I just use the sql data type name for the test and in DB its called `money`